.ci/travis-functions.sh

```diff
@@ -65,21 +64,18 @@ function xdebug_enable() {
 #   String
 #######################################
 function get_infection_pr_flags() {
-    local flags="--no-progress";
```

Comment:
this flag has been moved to the travis configuration instead: makes it less "hidden" and also ensures it is used even in a regular non-PR build